presto-testing/src/main/java/io/prestosql/testing/DistributedQueryRunner.java

```diff
@@ -186,12 +186,23 @@ private static TestingPrestoServer createTestingPrestoServer(
         ImmutableMap.Builder<String, String> propertiesBuilder = ImmutableMap.<String, String>builder()
```

Comment:
> Tests start multiple Presto nodes together (3 by default) 

It means actually 4. 3 workers and one coordinator. How about changing the default to 2 workers?

Reply:
i believe nodeCount=3 is the default and nodeCount=3 means 3 nodes: 1 coordinator and 2 workers